app/experimenter/experiments/tests/test_models.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 import os.path
 from decimal import Decimal
+from logging.config import valid_ident
```

Comment:
Did vscode throw that in?  Does it all the time to me, silly thing.

Reply:
I already deleted it hehe